pwa/src/templates/templateParts/mappingForm/MappingFormTemplate.tsx

```diff
@@ -248,11 +374,18 @@ export const MappingFormTemplate: React.FC<MappingFormTemplateProps> = ({ mappin
                   </div>
 
                   {isOpenCast && (
-                    <div className={styles.triggerButton} onClick={(e) => handleScrollToCreate(e, castRefBottom)}>
-                      <Link icon={<FontAwesomeIcon icon={faArrowDown} />} iconAlign="start">
-                        {t("Scroll to create")}
-                      </Link>
-                    </div>
+                    <>
+                      <div onClick={(e) => handleSwitchEditor(e, "unset")}>
```

Comment:
I think this should pass "cast" to the `handleSwitchEditor`, right now it's opening the wrong one.